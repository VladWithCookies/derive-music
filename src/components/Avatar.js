import * as React from 'react'

const Avatar = ({ image }) => (
  <img
    src={image}
    alt="Guy with a cup of coffee"
    className="object-cover w-40 h-40"
  />
)

export default Avatar
