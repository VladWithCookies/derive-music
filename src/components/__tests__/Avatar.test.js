import React from 'react'
import { render } from '@testing-library/react'

import Avatar from '../Avatar'

describe('Avatar', () => {
  it('matches snapshot', () => {
    const props = { image: 'image' };
    const { container } = render(<Avatar {...props} />)

    expect(container).toMatchSnapshot()
  })
})
