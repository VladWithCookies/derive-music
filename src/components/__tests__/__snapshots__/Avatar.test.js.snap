// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar matches snapshot 1`] = `
<div>
  <img
    alt="Guy with a cup of coffee"
    class="object-cover w-40 h-40"
    src="image"
  />
</div>
`;
